torch/_inductor/codegen/wrapper.py

```diff
@@ -2300,6 +2300,8 @@ def val_to_arg_str(self, val):
         if val is None:
             # When None is passed as an argument, it represents an optional that does not contain a value.
             # TODO: add abi-compatible support
+            if config.aot_inductor.abi_compatible:
+                return "0" # nullptr is not available in C
```

Comment:
Curious if `0` gets converted to `c10::nullopt` automatically when passed as argument for that type?

Reply:
It does not. FWIW, I like the explicitness of c10::nullopt anyways for its better readability.